debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/macho/ThreadCommand.java

```diff
@@ -132,7 +98,51 @@ public short readShort(int start)
 			return (endianness == ByteOrder.LITTLE_ENDIAN) ? ret : Short.reverseBytes(ret);
 		}
 
-		public void fillX86ThreadRegisters()
+		public abstract void fillThreadRegisters();
+		public abstract void fillExceptionData();
+
+	}
+
+	public static class OSXAMD64ThreadState extends ThreadState
```

Comment:
This class and `OSXAArch64ThreadState` should be `final`.

Reply:
Fixed.